### PR TITLE
Moving keyring note up to the right step

### PR DESCRIPTION
**Pull request by staticdev**
I am just moving it up since it is relevant to step 2, and I lost 15 minutes because I did not see the note, hopefully I will save time of other people.

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -185,6 +185,11 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+{{< note >}}
+In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
+you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
+{{< /note >}}
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:191]: Even better IMO (but I didn't test it):
```suggestion
   {{< note >}}
   In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
   you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
   {{< /note >}}
```

That makes the note as logically part the existing ordered list item.

**Comment by sftim** [comment]
NB. This is similar to https://github.com/kubernetes/website/pull/44636 (and I provided similar feedback there).

**Review by Princesso — COMMENTED** [review]
/lgtm

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (staticdev) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:191]: It also makes sense to me, but for soms reason (maybe a bug on this tabbed design) it seems to break. The way i commited works despite not perfectly aligned.

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -185,6 +185,11 @@ These instructions are for Kubernetes {{< skew currentVersion >}}.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+   {{< note >}}
+   In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default;
+   you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
+   {{< /note >}}
+
```
Comment (dipesh-rawat) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:192]: The page layout is broken due to this change _(indentation of the note block might be causing this)_. 
See the preview ([https://deploy-preview/tools/kubeadm/install-kubeadm/](https://deploy-preview-44671--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)) for more insights.

Comment (sftim) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:192]: Fair. A bit frustrating, but fair.

**Comment by sftim** [comment]
PR #44636 (which is similar) has merged already.

**Comment by staticdev** [comment]
Good, then I close this if favor to https://github.com/kubernetes/website/pull/44636

**Pull request closed without merging**